configs/components/gcc.rb

```diff
@@ -1,6 +1,6 @@
 component "gcc" do |pkg, settings, platform|
   # Source-Related Metadata
-  if platform.name =~ /fedora-f24|fedora-f25|ubuntu-16\.10/
+  if platform.name =~ /fedora-f24|fedora-f25|ubuntu-16\.04-ppc64el|ubuntu-16\.10/
```

Comment:
What are thoughts on revving our pl-gcc version to 6.1 for Ubuntu 16.04 for all arches (not just`ppc64el`)?